lib/service-manager.js

```diff
@@ -15,12 +15,41 @@ module.exports = ServiceManager;
 function ServiceManager(server) {
   this._server = server;
   this._meshApp = null;
+
+  // Bind handle to this for use as middleware.
+  this.handle = this.handle.bind(this);
 }
 
 util.inherits(ServiceManager, MeshServiceManager);
 
 var prototype = ServiceManager.prototype;
 
+// Support pm v3.x/mesh v5.x deployments, by directing all these requests to
+// service 'default' as deploy requests. Default will be created if it doen't
+// exist.
+prototype.handle = function(req, res, next) {
+  var url = req.url;
+
+  if (!/\/default$/.test(url) && !/\/default\/.*/.test(url))
+    return next();
+
+  debug('handle old-style deploy to: %s', req.url);
+
+  var Service = this._meshApp.models.ServerService;
+  var serviceName = 'default';
+  var scale = 1;
+
+  Service.serviceFindOrCreate(serviceName, scale, function(err, service) {
```

Comment:
As per my comment on the corresponding strong-mesh-models PR, this could be done without modifying strong-mesh-models:
```js
var filter = {where: {name: name}};
var service = {name: name, _groups: [{id: 1, name: 'default', scale: scale}]};
Service.findOrCreate(filter, service, function(err, service) {
  // ...
  service.deploy(req, res, next);
});
```